src/PlanView/PlanView.qml

```diff
@@ -710,6 +715,20 @@ Item {
                         enabled:            true
                         visible:            true
                         dropPanelComponent: centerMapDropPanel
+                    },
+                    ToolStripAction {
+                        text:               qsTr("Tools")
+                        iconSource:         "/qmlimages/MissionTools.svg"
+                        enabled:            _aviantSettings.missionToolsUrl.rawValue != ""
+                        visible:            true
+                        dropPanelComponent: missionToolsDropPanel
+                    },
+                    ToolStripAction {
+                        text:               qsTr("Validate")
+                        iconSource:         "/qmlimages/MissionValidation.svg"
+                        enabled:            _aviantSettings.missionToolsUrl.rawValue != "" && !_planMasterController.syncInProgress && _planMasterController.containsItems && _aviantMissionTools.missionType != AviantMissionTools.NotSet
```

Comment:
Requiring the mission type to be set for this button to work can be confusing when the mission type dropdown is hidden behind the "Tools" button. 
This applies to the validation functionality as well. It is not intuitive to me as a user that a dropdown inside a popup from the "Tools" button should affect the function of the "Validate" button. This can cause users to validate using a different mode than intended, leading to undiscovered mission errors.

Reply:
Will enable validate button even if type is not set, and add type selection to validate pane as well:
![image](https://user-images.githubusercontent.com/108630475/225265067-ec36663a-474c-4a95-969d-cf0784ea9255.png)
![image](https://user-images.githubusercontent.com/108630475/225265151-72b6648a-46db-4672-8eb1-66ee92e7755b.png)

Validation will be redone if type is changed.